project/src/screens/main/main.tsx

```diff
@@ -7,22 +7,44 @@ import MainEmpty from '../../components/main-empty/main-empty';
 import pluralize from 'pluralize';
 import {useState} from 'react';
 import {useAppDispatch, useAppSelector} from '../../hooks';
-import {changeCity} from '../../store/action';
+import {changeCity, changeSortType} from '../../store/action';
+import {SORT_TYPE} from '../../const';
+import {SortHighToLow, SortLowToHigh, SortTopRated} from '../../utils/utils';
 
 function Main(): JSX.Element {
   const cityName = useAppSelector((state) => state.city);
   const offers = useAppSelector((state) => state.offers);
+  const sortType = useAppSelector((state) => state.sortType);
   const dispatch = useAppDispatch();
 
   const currentOffers = offers.filter((offer) => offer.city.name === cityName);
   const isEmpty = currentOffers.length === 0;
 
+  const getSortOffers = () => {
+    switch (sortType) {
+      case SORT_TYPE[1]:
```

Comment:
Можно лучше: при использовании объекта будет видно ключи, что даст больше понимания
```suggestion
      case SortType.Popular:
```